apps/web/screens/Category/Category.tsx

```diff
@@ -276,7 +276,7 @@ const Category: Screen<CategoryProps> = ({
       >
         <Box paddingBottom={[2, 2, 4]}>
           <Breadcrumbs>
-            <Link href={pathNames()}>Ísland.is</Link>
+            <Link href={pathNames().as}>Ísland.is</Link>
```

Comment:
cant we spread pathnames here?
```suggestion
            <Link {...pathNames()}>Ísland.is</Link>
```